code/modules/admin/verbs/adminpm.dm

```diff
@@ -223,9 +223,10 @@
 	var/compliant_msg = trim(lowertext(msg))
 	var/irc_tagged = "[sender](IRC)"
 	var/list/splits = splittext(compliant_msg, " ")
+	var/usage_string = "Usage: ticket <close|resolve|icissue|reject|reopen \[ticket #\]|list>"

```

Comment:
Based on the fact that static vars name collide with all other statics/globals I'd rather not